src/resources/js/v2/tickets-commerce.js

```diff
@@ -30,6 +30,16 @@ tribe.tickets.commerce = {};
 	'use strict';
 	const $document = $( document );
 
+	/**
+	 * Ticket Commerce custom Events.
+	 *
+	 * @since TBD
+	 */
+	obj.customEvents = {
+		showLoader : 'tribeTicketsCommerceCheckout.showLoader',
+		hideLoader : 'tribeTicketsCommerceCheckout.hideLoader',
```

Comment:
Let's make sure we use `tecTicketsCommerce` no need for `Checkout` there too.
For jQuery events, the namespace comes after the event name. So it would be :
* `'showLoader.tecTicketsCommerce'`